### DISCUSSION TITLE
[aria-query] version 5 released, changed return types

### DISCUSSION THREAD

**Discussion by IanVS** [DefinitelyTyped/DefinitelyTyped#56549] [Issues with a @types package] [answered]
Version 5 removed the use of Map and Set: https://github.com/A11yance/aria-query/blob/main/CHANGELOG.md#500

I might try figuring out how to contribute to DefinitelyTyped, but I'm a bit intimidated by it, to be honest.

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment, marked as answer]
Will be updated by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56591
